pyriemann_qiskit/ensemble.py

```diff
@@ -0,0 +1,142 @@
+"""
+Ensemble classifiers.
+"""
+import numpy as np
+from pyriemann_qiskit.utils import union_of_diff
+from sklearn.base import ClassifierMixin, BaseEstimator
+
+
+class JudgeClassifier(BaseEstimator, ClassifierMixin):
+
+    """Judge classifier
+
+    Several classifiers are trained on the balanced dataset.
+    When at least one classifier disagrees on the label of a given input
+    in the training set, the input was noted.
+    These inputs, a subset of the training data of the balanced dataset,
+    formed an additional dataset on which a metaclassifier was subsequently

```

Comment:
What is the difference between classifier and metaclassifier?
```suggestion
    formed an additional dataset on which a new classifier was subsequently

```